### PR TITLE
ECR and queue

### PR DESCRIPTION
**Pull request by joakimmoller**
statemachine input is a list of jobs
batch lambda splits into FM-jobs
Jobs are queued to FM-queues
Fargate service processes jobs and scales on queue size and cpu usage 0-5 tasks
images are built and pushed in separate repo

### CODE REVIEW

File: level2/handlers/batch.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+from json import dumps
+from typing import Any, Dict, List, Literal, NewType
+from typing_extensions import TypedDict
+from Crypto.Cipher import AES
+from pydantic import TypeAdapter
+from boto3 import client
+
+ODIN_API_ROOT = "https://odin-smr.org"
+
+PROJECT_CONF = {
```
Comment (skymandr) [inline level2/handlers/batch.py:12]: Note to self: Here we will need to add 14, 22 and 24, at some point.

File: requirements.in [hunk 2]
```diff
@@ -1,5 +1,4 @@
 aws-cdk-lib
-aws-cdk.aws-lambda-python-alpha
 boto3
-requests
 pycryptodome
+pydantic
```
Comment (skymandr) [inline requirements.in:4]: What's this?

**Review by skymandr — APPROVED** [review]
LGTM

File: requirements.in (hunk 2 above)
Comment (joakimmoller) [reply requirements.in:4]: Provides very easy and in this case forgiving json parse to object functions.

**Pull request merged**